components/support/migration/src/main/java/mozilla/components/support/migration/FennecMigrator.kt

```diff
@@ -245,6 +262,16 @@ class FennecMigrator private constructor(
         return runMigrationsAsync(migrationsToRun)
     }
 
+    /**
+     * Performs the configured data migration blocking the current thread until it has been completed.
+     *
+     * @return A [MigrationResults], describing which migrations were performed and if they succeeded.
+     */
+    @CheckResult
+    fun migrateBlocking(): MigrationResults {
```

Comment:
I think I'd prefer to just do `runBlocking { migrateAsync.await() }` in Fenix instead. Not sure how much value this method really adds.

It'll give us more control over what's going closer to where we actually care about _how_ this runs (befoe gecko init in apps's onCreate).